integration-tests/src/slowIntegrationTest/java/pl/allegro/tech/hermes/integrationtests/HermesServerGracefulShutdownTest.java

```diff
@@ -0,0 +1,65 @@
+package pl.allegro.tech.hermes.integrationtests;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.web.reactive.server.WebTestClient;
+import pl.allegro.tech.hermes.frontend.server.HermesServer;
+import pl.allegro.tech.hermes.integrationtests.client.FrontendTestClient;
+import pl.allegro.tech.hermes.integrationtests.setup.HermesFrontendTestApp;
+import pl.allegro.tech.hermes.integrationtests.setup.InfrastructureExtension;
+import pl.allegro.tech.hermes.test.helper.message.TestMessage;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.awaitility.Awaitility.waitAtMost;
+import static pl.allegro.tech.hermes.frontend.FrontendConfigurationProperties.FRONTEND_GRACEFUL_SHUTDOWN_ENABLED;
+
+public class HermesServerGracefulShutdownTest  {
+
+    @RegisterExtension
+    public static InfrastructureExtension infra = new InfrastructureExtension();
+
+    private HermesFrontendTestApp frontend;
+    private HermesServer hermesServer;
+    FrontendTestClient frontendClient;
+
+    @BeforeEach
+    public void beforeEach() {
+        frontend = new HermesFrontendTestApp(infra.hermesZookeeper(), infra.kafka(), infra.schemaRegistry());
+        frontend.withProperty(FRONTEND_GRACEFUL_SHUTDOWN_ENABLED, false);
```

Comment:
If I'm not mistaken, this parameter doesn't affect the tests in this class in any way. Yet it is a bit misleading to set it to true while we are testing that the graceful shutdown is enabled. Perhaps, we can skip setting it.

Reply:
agree, I've set it by default as it was in old e2e tests and removed it from this test